src/install.rst

```diff
@@ -24,316 +24,266 @@ The following instructions describe how to install Nextstrain's software tools,
 
-      6. Create a Conda environment named ``nextstrain``:
+         .. group-tab:: Ubuntu Linux
 
-         .. include:: snippets/conda-create-powershell.rst
+            .. note:: Steps for other Linux distributions (Debian, CentOS, RHEL, etc.) will be similar, though specific commands may vary slightly.
 
-      7. Install the Nextstrain CLI:
+            1. Install Docker Engine for Ubuntu using the `convenience script <https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script>`_:
 
-         .. code-block:: powershell
+               .. code-block:: bash
 
-            mamba install --yes nextstrain-cli
+                  curl -fsSL https://get.docker.com -o get-docker.sh
+                  sudo sh get-docker.sh
+                  # follow through installation prompts
+                  rm get-docker.sh
 
-      8. Confirm that the installation worked.
+               .. note::
 
-         .. code-block:: powershell
+                  Alternatively, you can install Docker Engine using your system's package manager:
 
-            nextstrain check-setup --set-default
+                  .. code-block:: bash
 
-         The final output from the last command should look like this:
+                     sudo apt install docker.io
 
-         .. code-block:: none
+            2. Add your user to the `docker` group:
 
-            Setting default environment to docker.
+               .. code-block:: bash
 
-   .. group-tab:: WSL on Windows
+                  sudo gpasswd --add $USER docker
 
-      .. note::
+            3. Restart your machine.
+            4. Install the Nextstrain CLI:
 
-         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for Augur <augur:installation/installation>` and :doc:`install Auspice via npm <auspice:introduction/install>`.
+               .. code-block:: bash
 
-      1. `Install Windows Subsystem for Linux (WSL) 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
+                  curl -fsSL --proto '=https' https://nextstrain.org/install/cli/linux | bash
 
-         .. note:: You may have to restart your machine when configuring WSL.
 
-      2. Open a WSL terminal by running **wsl** from the Start menu.
-      3. Install Miniconda:
+   .. group-tab:: Native
 
-         .. code-block:: bash
+      .. tabs::
 
-            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-            bash Miniconda3-latest-Linux-x86_64.sh
-            # follow through installation prompts
-            rm Miniconda3-latest-Linux-x86_64.sh
+         .. group-tab:: macOS
 
-      4. Install Mamba on the ``base`` Conda environment:
+            .. warning::
 
-         .. code-block:: bash
+               If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), first run these commands to ensure Conda creates the environment with ``osx-64`` emulation:
```

Comment:
This no longer makes sense in its new location given that installation of Miniconda (`conda`) doesn't happen now until *after* this warning. Need to re-arrange, but also, @victorlin has a better alternative suggestion of using the x86_64 installer for Miniconda always, even on M1 hardware.

Reply:
Resolved by rebasing this PR onto the latest master and incorporating the changes there addressing this since I initially branched.